data/report_templates/project_summary.html

```diff
@@ -396,42 +410,32 @@
   <div class="clearfix"></div>
-  National Genomics Infrastructure - $title$
+  National Genomics Infrastructure - {{ get(meta.title) }}
 </div>
 
 <header>
-$if(subtitle)$<h1 class="title">$title$</h1>$endif$
-$if(subtitle)$<h1 class="subtitle">$subtitle$</h1>$endif$
-$if(date)$<h3 class="date">$date$</h3>$endif$
+{% if meta.subtitle %}<h1 class="title">{{ get(meta.title) }}</h1>{% endif %}
```

Comment:
Looks like an error in the original code too..
```suggestion
{% if meta.title %}<h1 class="title">{{ meta.title }}</h1>{% endif %}
```
Don't think you need the `get()` as you're already checking that it is set..

Reply:
Same for the` get()` here.